From c3bf5e39469d09667939155912ce4f83dfc1da41 Mon Sep 17 00:00:00 2001
From: Josh Triplett <josh@joshtriplett.org>
Date: Mon, 23 Dec 2013 13:55:19 -0800
Subject: [PATCH 4/7] staging: rts5139: Drop print of build time

Patch-mainline: 3.14-rc1
Git-commit: 0fcaf3bc1a12d4b21d4b215f831a7bc49baf6f04

The kernel already has this information, and individual drivers
shouldn't duplicate that.  This also eliminates the use of __TIME__,
which makes the build non-deterministic.  (And, without __DATE__,
__TIME__ provided little useful information to begin with.)

Signed-off-by: Josh Triplett <josh@joshtriplett.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Jan Moskyto Matejka <mq@suse.cz>
---
 drivers/staging/rts5139/rts51x_scsi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/staging/rts5139/rts51x_scsi.c b/drivers/staging/rts5139/rts51x_scsi.c
index 6108705..dd86b85 100644
--- a/drivers/staging/rts5139/rts51x_scsi.c
+++ b/drivers/staging/rts5139/rts51x_scsi.c
@@ -1985,7 +1985,6 @@ static int show_info(struct seq_file *m, struct Scsi_Host *host)
 	SPRINTF("       Vendor: Realtek Corp.\n");
 	SPRINTF("      Product: RTS51xx USB Card Reader\n");
 	SPRINTF("      Version: %s\n", DRIVER_VERSION);
-	SPRINTF("        Build: %s\n", __TIME__);
 	return 0;
 }
 
-- 
1.8.4.5

